service-front/module/Application/view/application/pages/post_office/donor_lpa_check.twig

```diff
@@ -1,8 +1,9 @@
+{% extends "layout/layout" %}
+{% block content %}
 {% include 'layout/id_check_banner.twig' with details_data %}
 <div class="govuk-width-container govuk-!-padding-top-5">
     <form method="POST">
-        <h1 class="govuk-heading-xl">Donor LPA check</h1>
-        <h3 class="govuk-heading-m">Which LPAs should this identity check apply to?</h3>
+        <h3 class="govuk-heading-m">{% block title %}Which LPAs should this identity check apply to?{% endblock %}</h3>
```

Comment:
For semantics, this should be an h1. It can still be `govuk-heading-m` for style/sizing.